examples/reference/panes/HoloViews.ipynb

```diff
@@ -119,18 +207,205 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "widgets = hv_panel[1]\n",
+    "plot = df.hvplot.scatter(x=\"group\", y=\"value\")\n",
+    "pn.pane.HoloViews(plot, backend='matplotlib', sizing_mode=\"stretch_width\", height=300)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Please note that in a *server context* you might have to set the matplotlib backend like below depending on your setup.\n",
+    "\n",
+    "```python\n",
+    "import matplotlib\n",
+    "matplotlib.use('agg')\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can make your matplotlib plot stretch responsively by ???? **DON'T KNOW HOW TO. PLEASE HELP**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Plotly\n",
+    "\n",
+    "To use the 'plotly' plotting backend you will need to run `hv.extension(\"plotly\")` to configure the 'plotly' backend.\n",
+    "\n",
+    "If you are using `hvPlot` you can use `hvplot.extension(\"plotly\")` instead."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hvplot.extension(\"plotly\")\n",
+    "\n",
+    "plot = df.hvplot.scatter(x=\"group\", y=\"value\", height=300, responsive=True)\n",
+    "pn.pane.HoloViews(plot, backend='plotly', height=300)"
```

Comment:
There is a **bug** here. The resulting plot is higher than the `300px` specified both with hvPlot and HoloViews.

![image](https://github.com/holoviz/panel/assets/42288570/55d1ed8e-0146-4d25-8050-5f9f713cb988)